test

```diff
@@ -0,0 +1,19 @@
+echo CALICOCTL
```

Comment:
Oops, I made a commit right before I left yesterday that removed it, but pushed to metaswitch/calico-docker instead of djosborne/calico-docker, which is why that new branch was accidently created.

I cherry-picked the commit, then removed the branch for the metaswitch repo.

I think I know how to squash this into one commit, I'll give it a shot when I get to the office and then ping you with how it goes.